src/SprykerEco/Yves/Payone/Form/DataProvider/PostfinanceEfinanceOnlineTransferDataProvider.php

```diff
@@ -47,7 +47,7 @@ public function getOptions(AbstractTransfer $quoteTransfer)
     /**
      * @return array
```

Comment:
```suggestion
     * @return array<string, string>
```

Please check other return values like this as well

Reply:
Fixed.